src/table/control_codes.h

```diff
@@ -27,8 +27,10 @@ enum StringControlCode {
 	SCC_ENCODED = SCC_CONTROL_START,
 
 	/* Display control codes */
-	SCC_TINYFONT,  ///< Switch to small font
-	SCC_BIGFONT,   ///< Switch to large font
+	SCC_NORMALFONT, ///< Switch to normal size font
```

Comment:
Add a comment that the font codes should be kept in order?

Reply:
I'll add it but there are lots of other enums lacking that comment already.

Reply:
Actually, it should probably have extra SCC_FONT_START/END constants but I can't even find a good example for than in other code.

Reply:
I think just a comment just before the font codes would suffice, don't worry about other enums :-)